### DISCUSSION TITLE
ERR_SSL_KEY_USAGE_INCOMPATIBLE on SO version 2.3.300 and open with chrome v.126

### DISCUSSION THREAD

**Discussion by golfreeze** [Security-Onion-Solutions/securityonion#13302] [General]
![pic](https://github.com/Security-Onion-Solutions/securityonion/assets/3982241/1e262b7c-347b-407c-a11c-286e10cf98d5)
ERR_SSL_KEY_USAGE_INCOMPATIBLE on SO version 2.3.300 and open with chrome v.126

**Comment by dougburks** [comment]
Please note that Security Onion 2.3 has reached End Of Life and is no longer supported:
https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html

Please migrate to Security Onion 2.4. In the meantime, you might try using a different web browser.

**Comment by golfreeze** [comment]
Hi Dougburks and team,

Thank you for your suggestion.

I've try openssl gen with KeyUsage:digitalSignature, keyEncipherment, nonRepudiation
this step and could use chrome version 126 ,

===== Start ======
openssl req -x509 -nodes -newkey rsa:4096 -keyout server.key -out server.crt -days 7300 -subj '/CN=pklteam/C=US/L=Salt Lake City/ST=Utah' -addext "keyUsage = digitalSignature, keyEncipherment, nonRepudiation" -addext "extendedKeyUsage = serverAuth, clientAuth" -addext "subjectAltName=DNS:pklteam,IP:150.xx.yy.zz"

change IP: as your IP address 
and DNS: as your hostname

== backup managerssl.*
mkdir -p /etc/pki/backup_cert 
cp /etc/pki/managerssl.* /etc/pki/backup_cert/.
== copy new cert and key to managerssl.*
cp server.crt /etc/pki/managerssl.crt
cp server.key /etc/pki/managerssl.key
==restart nginx 1 times 
so-nginx-restart

==testing on chrome access -> Working well.
